apps/frontend/src/components/forms/StartNewProjectForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { FormattedMessage } from 'react-intl';
+
+export interface StartNewProjectFormProps {
+  onSubmit: SubmitHandler<StartNewProjectFormData>;
+}
+export interface StartNewProjectFormData {
+  name: string;
+  mapStyle: string;
+  template?: string;
+}
+
+export function StartNewProjectForm({ onSubmit }: StartNewProjectFormProps) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<StartNewProjectFormData>();
+
+  return (
+    /* "handleSubmit" will validate your inputs before invoking "onSubmit" */
+    <form onSubmit={handleSubmit(onSubmit)}>
+      {/* register your input into the hook by invoking the "register" function */}
+      <input defaultValue="" {...register('name', { required: true })} />
+      {errors.name && <FormattedMessage defaultMessage="This field is required" />}
+
+      <select {...register('mapStyle')}>
+        <option value="satellite">satellite</option>
+        <option value="topo">topo</option>
+      </select>
+
```

Comment:
There miss the template select in the form. 
It's could be interesting to test the card design ?

Reply:
Absolutely ! You'll add your components to this form once this is merged.